### DISCUSSION TITLE
Updated vue demo writing specifications

### DISCUSSION THREAD

**Discussion by xxholly32** [element-plus/element-plus#16767] [Ideas]
There has been a noticeable change to setup in the [vue playground](https://play.vuejs.org/), which is to move script tag to the top.

Although the best practices of the official vue team don't seem to be out yet, the community is leaning toward doing so

https://www.zhihu.com/question/483860485/answer/2097414934

**Comment by warmthsea** [comment]
👀  Maybe it can be handled by eslint with one click.

**Reply by xxholly32** [reply]
yes. In my project, I can introduce @antfu/eslint-config to solve the problem, but mainly consider whether the code specification and element-plus source code in the official website demo are also necessary to adjust
